Homework4/ParseTree/ParseTree/Operand.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace ParseTree;
+
+class Operand : INode
+{
+    private double value;
+    public Operand(double newValue)
```

Comment:
```suggestion
    private double value;
    
    public Operand(double newValue)
```
Тут и ниже пустых строк сильно не хватает